### PR TITLE
[R2] Runtime APIs revision

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: content/r2/runtime-apis.md [hunk 1]
```diff
@@ -60,30 +59,30 @@ async function handleRequest(request) {
...
 
-  - Retrieves the R2Object for the given key containing only object metadata, if the key exists, and null if the key does not exist.
+  - Retrieves the `R2Object` for the given key containing only object metadata, if the key exists, and null if the key does not exist.
 
 - {{<code>}}get(key{{<param-type>}}string{{</param-type>}}, options{{<param-type>}}R2GetOptions{{</param-type>}}{{<prop-meta>}}optional{{</prop-meta>}}) {{<type>}}Promise\<{{<param-type>}}R2Object{{</param-type>}}|{{<param-type>}}null{{</param-type>}}>{{</type>}}{{</code>}}
 
-  - Retrieves the R2Object for the given key containing object metadata and the object body as a {{<code>}}{{<param-type>}}ReadableStream{{</param-type>}}{{</code>}}, if the key exists, and null if the key does not exist.
+  - Retrieves the `R2Object` for the given key containing object metadata and the object body as a {{<code>}}{{<param-type>}}ReadableStream{{</param-type>}}{{</code>}}, if the key exists, and null if the key does not exist.
```
Comment (marciocloudflare) [inline content/r2/runtime-apis.md:66]: Should null be in between backticks?
```suggestion
  - Retrieves the `R2Object` for the given key containing object metadata and the object body as a {{<code>}}{{<param-type>}}ReadableStream{{</param-type>}}{{</code>}}, if the key exists, and null if the key does not exist.
```

Comment (marciocloudflare) [inline content/r2/runtime-apis.md]:
```suggestion
  - Retrieves the `R2Object` for the given key containing object metadata and the object body as a {{<code>}}{{<param-type>}}ReadableStream{{</param-type>}}{{</code>}}, if the key exists, and `null` if the key does not exist.
```

File: content/r2/runtime-apis.md [hunk 2]
```diff
@@ -200,23 +199,23 @@ R2GetOptions accepts a range parameter, which restricts data returned in {{<code
...
 
 - {{<code>}}include{{<param-type>}}Array\<string\>{{<prop-meta>}}optional{{</prop-meta>}}{{</param-type>}}{{</code>}}
-  - Can include `httpMetadata` and/or `customMetadata`.  If included, items returned by the list will include the specified metadata.
+  - Can include `httpMetadata` and/or `customMetadata`. If included, items returned by the list will include the specified metadata.
 
   - Note that there is a limit on the total amount of data that a single `list` operation can return. If you request data, you may recieve fewer than `limit` results in your response to accomodate metadata.
 
-  This means applications must be careful to **avoid** code like the following:
+  This means applications must be careful to avoid code like the following:
   ```js
```
Comment (marciocloudflare) [inline content/r2/runtime-apis.md:220]:
```suggestion

  ```js
```

File: content/r2/runtime-apis.md [hunk 3]
```diff
@@ -276,11 +275,11 @@ You can pass an R2Conditional object to R2GetOptions.  If the condition check fa
 
  {{</definitions>}}
 
-For more information about conditional requests, see RFC 7232.
+For more information about conditional requests, refer to RFC 7232. <!-- what is this?>
```
Comment (deadlypants1973) [inline content/r2/runtime-apis.md]: @lukeed

Comment (marciocloudflare) [inline content/r2/runtime-apis.md]: Did you notice there's a `<!-- what is this?>` still here?

**Review by marciocloudflare — APPROVED** [review]

File: content/r2/runtime-apis.md (hunk 3 above)
Comment (deadlypants1973) [inline content/r2/runtime-apis.md]:
```suggestion
For more information about conditional requests, refer to [RFC 7232](https://datatracker.ietf.org/doc/html/rfc7232).
```

**Pull request merged**